pkg-py/src/querychat/querychat.py

```diff
@@ -233,6 +237,74 @@ def df_to_html(df: IntoFrame, maxrows: int = 5) -> str:
     return table_html + rows_notice
 
 
+def _get_client_from_env() -> Optional[str]:
+    """Get client configuration from environment variable."""
+    env_client = os.getenv("QUERYCHAT_CLIENT", "")
+    if not env_client:
+        return None
+    return env_client
+
+
+def _create_client_from_string(client_str: str) -> chatlas.Chat:
+    """Create a chatlas.Chat client from a provider-model string."""
+    provider, model = (
+        client_str.split("/", 1) if "/" in client_str else (client_str, None)
+    )
+    # We unset chatlas's envvars so we can listen to querychat's envvars instead
+    with temp_env_vars(
+        {
+            "CHATLAS_CHAT_PROVIDER": provider,
+            "CHATLAS_CHAT_MODEL": model,
+            "CHATLAS_CHAT_ARGS": os.environ["QUERYCHAT_CLIENT_ARGS"],
+        },
+    ):
+        return chatlas.ChatAuto(provider="openai")
```

Comment:
Just curious, did you have a look into https://github.com/posit-dev/chatlas/issues/90#issuecomment-3168335019 and decide it wasn't worth doing (for now)?

Reply:
No, I think it's very worth doing. I was just trying to set this up here so that it's not blocked by that work, but in a way that it wouldn't require many changes when we get to posit-dev/chatlas#90.